packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -169,16 +174,18 @@ export function buildRootObject() {
             ...oldAllocation,
             ...updates,
           });
-          const zcfSeatAdmin = seatToZCFSeatAdmin.get(zcfSeat);
+          const zcfSeatAdmin = zcfSeatToZCFSeatAdmin.get(zcfSeat);
           // verifies offer safety
           const seatStaging = zcfSeat.stage(newAllocation);
           // No effects above. Commit point. The following two steps
           // *should* be committed atomically.
```

Comment:
I don't thing "the following two steps" is still apropos.  Is this PR the atomic commitment that the comment was hoping for?

Reply:
Just to clarify, I think "the following two steps" doesn't refer to the next two lines, but instead to `.commit()`, and `.replaceAllocations()`. It would be nice to expand the description to say which steps are intended.